encrypted_json_fields/utils.py

```diff
@@ -0,0 +1,18 @@
+
+
+def pad(data: bytes, block_size: int) -> bytes:
```

Comment:
`Crypto.Util.Padding` provides pad and unpad methods.

```
padded_data = pad(plaintext, AES.block_size)

```

Reply:
I'll add that package